index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Inter:wght@100;200;300;400;500;600;700;800;900&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="style.css" />
+    <title>Portfolio</title>
+  </head>
+
+  <body>
+    <header class="container">
+      <p id="logo-placeholder">Welcome</p>
+      <img src="images/burger-menu.png" alt="" />
+    </header>
+
+    <section id="headline" class="container">
+      <h1>
+        Hey there. I’m Hayet <br />
+        <span class="color-green">I’m a software developer</span>
+      </h1>
+      <p>
+        I can help you build a product, feature or website Look through some of
+        my work and experience! If you like what you see and have a project you
+        need coded, don’t hesitiate to contact me.
+      </p>
+
+      <div id="social-media" class="container">
+        <a href="https://twitter.com/hayet_fe"> <img src="images/twitter.png" alt="twiiter icon" class="icon" /></a>
+        <a href="https://www.linkedin.com/in/hayet-f-5b4347247/"><img src="images/linkedin.png" alt="linkedin icon" class="icon" /></a>
+        <a href="#"> <img src="images/medium.png" alt="medium icon" class="icon" /></a>
+        <a href="https://github.com/laiifuu"><img src="images/github.png" alt="github icon" class="icon" /></a>
+        <a href="#"><img src="images/angellist.png" alt="angellist icon" class="icon" /></a>
+      </div>
```

Comment:
Please make sure that the list of icons uses a `ul` tag to group them, and `li` per icon; each icon has a link (`a` tag) 🤗

- _Apart from being semantically correct, the `a` tag will let you connect your social networks with the icons, using href._